### ISSUE TITLE
Dart SDK warning in Atom

### ISSUE THREAD

**Issue by mit-mit** [dartino/sdk#487]
After installing the Dartino plugin, I get a warning prompt saying:

```
SDK version 1.14.2 is older than the recommended version of 1.15.0. 
Please visit www.dartlang.org to download a recent SDK.

Using SDK at /home/dartino/dartino-sdk/internal/dart-sdk.
```

**Comment by mit-mit** [comment]
This will likely go away once we fix #489

**Comment by danrubel** [comment]
Fixed https://github.com/dart-atom/dartlang/pull/981
